steps/changeownershipofwinpestagingdirectorytoattuneonmacosorlinuxworker/script.txt

```diff
@@ -0,0 +1,13 @@
+if {kickstartedBootLoaderIsUefi}
+then
+
+    cd "{automationWorkerLinuxBaseDirectory}/build-winpe-iso"
+    
+    USER="{automationWorkerLinuxUser.user}"
+    GROUP="$(id -gn {automationWorkerLinuxUser.user})"
+    echo "User: $USER, Group: $GROUP"
+    
+    chown -R -v "$USER:$GROUP" "winpe_staging"
+else

```

Comment:
This needs to be elif {kickstartedBootLoaderIsBios}

the else needs to throw an error